package.json

```diff
@@ -62,6 +62,7 @@
     "gatsby-plugin-emotion": "4.1.21",
     "gatsby-plugin-google-analytics": "2.1.33",
     "gatsby-plugin-layout": "1.1.21",
+    "gatsby-plugin-linkedin-insight": "^1.0.1",
```

Comment:
Can you keep this as an exact version number? Trying to stick to exact version numbers to avoid breaking builds without us explicitly changing dependencies.